docs/index.html

```diff
@@ -1 +1,204 @@
+                    <polyline class="a" points="7930.4,5495.7 5426.1,8000 7930.4,10504.3 "></polyline>
+                    <line class="a" x1="10573.9" y1="8000" x2="5426.1" y2="8000"></line>
+                </svg>
+            </div>
+            <div data-u="arrowright" class="jssora106" style="width:55px;height:55px;top:162px;right:30px;" data-scale="0.75">
+                <svg viewbox="0 0 16000 16000" style="position:absolute;top:0;left:0;width:100%;height:100%;">
+                    <circle class="c" cx="8000" cy="8000" r="6260.9"></circle>
+                    <polyline class="a" points="8069.6,5495.7 10573.9,8000 8069.6,10504.3 "></polyline>
+                    <line class="a" x1="5426.1" y1="8000" x2="10573.9" y2="8000"></line>
+                </svg>
+            </div>
+        </div>
+      </div>
+      <!-- /img slider row -->
+
+      <div class="row">
+        <div class="col-sm-4 my-4">
+          <div class="card">
+            <img class="card-img-top" src="https://raw.githubusercontent.com/RITlug/ritlug.github.io/master/img/ritlug-large.png" alt="Ritlug">
+            <div class="card-body">
+              <p class="card-text">The RIT Linux Users Group is an RIT-sponsored club for new and experienced users of the Linux operating system.</p>
+            </div>
+            <div class="card-footer">
+              <a href="http://ritlug.com/" class="btn btn-primary">RITLUG</a>
+            </div>
+          </div>
+        </div>
+        <div class="col-sm-4 my-4">
+          <div class="card">
+            <img class="card-img-top" src="img/Octocat.png" alt="">
+            <div class="card-body">
+              <h4 class="card-title">Github</h4>
+              <p class="card-text">GitHub brings teams together to work through problems, move ideas forward, and learn from each other along the way.</p>
+            </div>
+            <div class="card-footer">
+              <a href="https://github.com/RITlug/TigerOS" class="btn btn-primary">Find our project</a>
+            </div>
+          </div>
+        </div>
+        <div class="col-sm-4 my-4">
+          <div class="card">
+            <img class="card-img-top" src="https://opensource.org/files/osi_keyhole_300X300_90ppi_0.png" alt="">
+            <div class="card-body">
+              <h4 class="card-title">Open Source</h4>
+              <p class="card-text">TigerOS is a fully open source project, meaning the source code is publicly available for use and modification.</p>
+            </div>
+            <div class="card-footer">
+              <a href="https://opensource.org/" class="btn btn-primary">Find Out More!</a>
+            </div>
+          </div>
+        </div>
+
+      </div>
+      <!-- /.row -->
+
+    </div>
+    <!-- /.container -->
+    <!-- Footer -->
+    <footer class="py-5 bg-dark">
+      <div class="container">
+        <p class="m-0 text-center text-white">&copy; 2017 TigerOS</p>
```

Comment:
Instead of TigerOS for the copyright, can we make that RITlug?

Reply:
@jrtechs Okay, we decided on a Creative Commons Attribution-ShareAlike license for our website content, like text and screenshots. Please change `&copy;` to `<a href="https://creativecommons.org/licenses/by-sa/4.0/" target="_blank">CC-BY-SA 4.0</a>`

Thanks!